feat(subscription): add configurable heartbeat for subscription callback protocol (#4246)
apollographql/router: .changesets/feat_bnjjj_feat_4115.md

### feat(subscription): add configurable heartbeat for subscription callback protocol ([Issue #4115](https://github.com/apollographql/router/issues/4115))

> Add ability to disable heartbeat for architecture based on lambdas for example

Reviewer: ```suggestion
Add the ability to disable heartbeat for certain platforms; lambdas for example
```
